src/IniFile.cpp

```diff
@@ -893,6 +893,26 @@ void cIniFile::RemoveBom(AString & a_line) const
 
 
 
+AString cIniFile::TrimSpaces(const AString & a_String)
```

Comment:
You can just use the [TrimString](https://github.com/cuberite/cuberite/blob/master/src/StringUtils.cpp#L219) function.